exercises/crypto-square/example.el

```diff
@@ -0,0 +1,32 @@
+(defun normalize-text (string)
```

Comment:
I think we should include the same commentary in the example solution as we do in the solution template.

Reply:
Fixed